relayer-cli/.env.dist

```
@@ -6,12 +6,20 @@ PRIVATE_KEY=
 SENDER_ADDRESSES_DEVNET=0x906dE43dBef27639b1688Ac46532a16dc07Ce410,0x123456789abcdef123456789abcdef1234567890
 SENDER_ADDRESSES_TESTNET=0x0000000000000000000000000000000000000000
 SENDER_ADDRESS_MAINNET=
+# Hashi sender addresses, zero address to allow all senders and enable hashi executor
+SENDER_ADDRESSES_HASHI=0x0000000000000000000000000000000000000000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SENDER_ADDRESSES_HASHI key should go before the SENDER_ADDRESSES_TESTNET key

(UnorderedKey)